### ISSUE TITLE
Widget doesn't seem to show up

### ISSUE THREAD

**Issue by oddharsh** [melchor629/Playbox.widget#2]
I'm on Mojave on my 2014 MacBook Pro, and can't seem to see any popup (even the error popup from the coffee file). I tried installing the launch daemon and restarting, or using iTunes instead of Spotify. I was previously running the original version of the widget with no issue on the same setup.

**Comment by melchor629** [comment]
Hi, try running this command on the terminal `curl http://localhost:45987`. Check if there's any errors on the "Debug Console" of Übersicht too.

Also, what macOS version are you running on?

**Comment by oddharsh** [comment]
Hey, so after updating to the latest version of the widget, it does seem to show up and accurately show the progress of the track from spotify, but when I change the track, the widget doesn't refresh the artwork or display the banner indicating the new song's title, though the progress does display correctly. Thanks so much for updating the original by the way!

**Comment by melchor629** [comment]
Yes, I noticed this bug some days ago but I didn't upload the fix. Try now with the new version, it should fix those issues.

**Comment by oddharsh** [comment]
Thank you for updating the widget, but the problem seems to persist, even after reinstalling the daemon and restarting my machine. I'm using the stock settings on the latest version of ubersicht on macos mojave

**Comment by melchor629** [comment]
Uhm that's strange.

Try replacing the `GetCurrentTrack` binary (located in `Playbox.widget/lib/`, you will find it quick) with this build [GetCurrentTrack.zip](https://github.com/melchor629/Playbox.widget/files/2568384/GetCurrentTrack.zip) and then run (with a terminal) these commands (or log out an log in again):

```
launchctl unload -w ~/Library/LaunchAgents/me.melchor9000.getcurrenttrack.plist
launchctl load -w ~/Library/LaunchAgents/me.melchor9000.getcurrenttrack.plist
```

The commands what it does is restart the background service, so you don't need to log out and re-login or even restart the computer to apply an update to the service :)

Tell me if it fixes the issue, and I hope so.

**Comment by oddharsh** [comment]
The problem mostly persists, though the track in the info popup refreshes (until the popup hides, at which point it remains broken). This might just be an issue with my computer, so I might just switch back to the original version of Playbox. Thanks for your help! :)

**Comment by oddharsh** [comment]
Also, is the react version of the widget ready to be formally released yet? I tried using the version you put in the last commit and got a error message.

**Comment by melchor629** [comment]
No, react version is not ready yet. I hope to be soon stable enough :)
